src/tools/autocompletion/find_autocompletion_options.sh

```diff
@@ -0,0 +1,9 @@
+#/usr/bin/env bash
+
+_helloworld_completions() {
+	# completing for last input before a tab
+	OUTPUT="$(python3 find_autocompletion_options.py ${COMP_WORDS[-1]})"
```

Comment:
```suggestion
	OUTPUT="$(python3 find_autocompletion_options.py spec/autocomplete ${COMP_WORDS[-1]})"
```

here you could pass the path to the specification

Reply:
I have added `-m spec/tests/autocomplete`